src/Slider.js

```diff
@@ -535,14 +561,16 @@ export default class Slider extends PureComponent<SliderProps, SliderState> {
           style={[defaultStyles.touchArea, touchOverflowStyle]}
           {...this._panResponder.panHandlers}
         />
-        <Bubble 
+       <Bubble 
           ref={(bubble: any) => this._bubble = bubble}
           value={this.props.value}
           thumbTintColor={thumbTintColor}
-          style={{
+          style={[{
             position: 'absolute',
             transform: [{ translateX: thumbLeft }, { translateY: 0 }],
-          }}
+          }, 
+          this.props.value < 1 ? {display: 'none'} : {},
```

Comment:
what 1 mean? can you move it to constant with a meaningful name?

Reply:
ok